jerry-core/ecma/builtin-objects/ecma-builtin-intrinsic.c

```diff
@@ -16,12 +16,28 @@
 #include "ecma-builtins.h"
 #include "ecma-array-object.h"
 #include "ecma-gc.h"
+#include "lit-char-helpers.h"
 
 #if ENABLED (JERRY_ES2015)
 
 #define ECMA_BUILTINS_INTERNAL
 #include "ecma-builtins-internal.h"
 
+/**
+ * This object has a custom dispatch function.
+ */
+#define BUILTIN_CUSTOM_DISPATCH
+
+/**
+ * List of built-in routine identifiers.
+ */
+enum
+{
+  ECMA_INTRINSIC_PARSE_FLOAT = ECMA_BUILTIN_ID__COUNT,
```

Comment:
I'd add the first entry to be consistent with the other custom dispatchers like::

```
ECMA_INTRINSIC_ROUTINE_START = ECMA_BUILTIN_ID__COUNT - 1,
```
after that `ECMA_INTRINSIC_PARSE_FLOAT`, is incremented automatically. ,